specification/appconfiguration/data-plane/Microsoft.AppConfiguration/stable/1.0/appconfiguration.json

```diff
@@ -1333,6 +1211,49 @@
       "type": "string",
       "x-ms-skip-url-encoding": true,
       "x-ms-parameter-location": "client"
+    },
+    "SelectSettings": {
+      "name": "$Select",
+      "in": "query",
+      "description": "Used to select what fields are present in the returned resource(s).",
+      "type": "array",
+      "items": {
+        "type": "string",
+        "enum": [
+          "key",
+          "label",
+          "content_type",
+          "value",
+          "last_modified",
+          "tags",
+          "locked",
+          "etag"
+        ],
+        "x-ms-enum": {
+          "name": "SettingFields",
```

Comment:
I would suggest `KeyValueFields` since the object we are selecting fields for is the `KeyValue` object. Json reference name as well.